data_structures/src/transaction.rs

```diff
@@ -430,8 +430,9 @@ pub struct TallyTransaction {
     pub dr_pointer: Hash, // DTTransaction hash
     // Outputs
     pub tally: Vec<u8>,
-    pub outputs: Vec<ValueTransferOutput>, // Witness rewards
-    pub slashed_witnesses: Vec<PublicKeyHash>,
+    pub outputs: Vec<ValueTransferOutput>,    // Witness rewards
```

Comment:
Please put this comment in a separated line, and add a comment to the `error_committers` field.

```suggestion
    /// Witness rewards
    pub outputs: Vec<ValueTransferOutput>,
```